packages/firestore/test/unit/util/bundle.test.ts

```diff
@@ -17,17 +17,38 @@
 import { expect } from 'chai';
 import {
   BundleReader,
-  SizedBundleElement,
-  toReadableStream
+  SizedBundleElement
 } from '../../../src/util/bundle_reader';
-import { isNode } from '../../util/test_platform';
 import { BundleElement } from '../../../src/protos/firestore_bundle_proto';
+import { isNode } from '../../util/test_platform';
+import {
+  PlatformSupport,
+  toByteStreamReader
+} from '../../../src/platform/platform';
 
-function readableStreamFromString(
+/**
+ * Create a `ReadableStream` from a string.
+ *
+ * @param content: Bundle in string.
+ * @param bytesPerRead: How many bytes to read from the underlying buffer from
+ * each read through the stream.
+ */
+export function readableStreamFromString(
   content: string,
-  bytesPerRead: number
-): ReadableStream {
-  return toReadableStream(new TextEncoder().encode(content), bytesPerRead);
+  bytesPerRead = 10240
+): ReadableStream<Uint8Array | ArrayBuffer> {
```

Comment:
Would it be possible to convert the input to a `UInt8Array` and re-use toByteStreamReader?